src/operators/ops_config.cpp

```diff
@@ -301,7 +302,7 @@ namespace
             if (res.has_value())
             {
                 frame f(runtime.default_value_scope(), res.value(), std::make_shared<behavior_configclasses_exit>(nav));
-                f["_x"] = nav->operator[](0);
+                f["_x"] = { *(nav.begin()) };
```

Comment:
This feels wrong somehow, but I'm not sure what it should be
Without it the first iteration had an invalid `_x`

Reply:
This is more about bad API design here sadly.
Need to do in-depths check for what is going on exactly tho to redesign that part of the class.

Feel free however to suggest something btw. :) i am open for change in that regard